src/cn-toast.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+  'use strict';
+
+  angular.module('cn.ui')
+      .controller('toastController', [
+        '$rootScope', 'toaster',
+        function($rootScope, toaster) {
+          const defaults = {
+            timeout: 5000,
+            toasterId: 'simple',
+            body: 'simple-toast',
+            bodyOutputType: 'directive',
+            tapToDismiss: false,
+            closeHtml: '<a>Dismiss</a>',
+            directiveData: {
+              type: 'success'
+            }
+          };
+
+          $rootScope.$on("citizenNet:toastEvent", function(event, options) {
+            let toastOptions = _.cloneDeep(defaults);
+            if (_.isObject(options)) {
+              _.extend(toastOptions, options);
+            } else {
+              toastOptions.directiveData.body = options;
+            }
```

Comment:
Above lines could be updated to

`toastOptions.directiveData.body = {
  ...defaults,
  ...options
};`

You might need to add the "transform-object-rest-spread" babel plugin